cookbooks/aws-parallelcluster-config/templates/default/init/fetch_and_run.erb

```diff
@@ -56,6 +69,24 @@ function run_postinstall () {
   fi || error_exit "Failed to run postinstall"
 }
 
+function run_postupdate () {
+  stack_status=$(get_stack_status) || error_exit "Failed to get the stack status"
+
+  if [ "${stack_status}" != "UPDATE_IN_PROGRESS" ]; then
+    echo "postupdate called with CFN state: ${stack_status}, doing nothing"
+    exit 0
+  fi
+
+  if [ "${cfn_postupdate}" != "NONE" ]; then
+    file="${cfn_postupdate}"
+    if [ "${cfn_postupdate_args}" != "NONE" ]; then
+      download_run ${cfn_postupdate} "${cfn_postupdate_args[@]}"
+    else
+      download_run ${cfn_postupdate}
+    fi
+  fi || error_exit "Failed to run postupdate"
```

Comment:
`Failed to run post update hook`

Reply:
I have no strong feelings on this, I used this phrasing to align on what we do for the other custom actions, as we use the phrasing `Failed to run preinstall and `Failed to run postinstall`.

Reply:
Actually, I'll make this change here and then change the other two messages to match, in a separate PR, as `post update hook` is clearer.